requirements.txt

```diff
@@ -4,6 +4,6 @@ parse-accept-language
 jsondiff
 graphene
 graphql-server-core
-marshmallow
+marshmallow==3.2.2
```

Comment:
PLZ don't freeze to a specific version. If 3.2.2 is the minimal version required you can set `marshmallow>=3.2.2` If you have some troubles with newer version can you describe the problem and the marshmallow version?

Reply:
@kevinkhao The problem is now solved. Freeze is no more required.